### ISSUE TITLE
Py3.6 Update certifi, urllib3, idna 

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2588]
Post #2567 in testing branch we are no longer limited by Py2.7. It is proposed that we update these secondary dependencies of our direct **requests** dependency by unpinning their versions.

In this case, now we use Poetry, it is proposed that we remove the version constraint in our pyproject.toml
https://pypi.org/project/certifi/ Latest still 3.6 compatible.
https://pypi.org/project/idna/ Latest still 3.6 compatible.
https://pypi.org/project/urllib3/ Latest need 3.7.

**Comment by phillxnet** [comment]
Given we have no direct use of certifi - only indirect via Requests - it is further proposed that we remove our specification re explicit dependency entirely: give it is not a direct dependency of ours.
```
requests 2.27.1 Python HTTP for Humans.
├── certifi >=2017.4.17
├── charset-normalizer >=2.0.0,<2.1.0
├── idna >=2.5,<4
└── urllib3 >=1.21.1,<1.27
```

**Comment by FroggyFlox** [comment]
Sounds like a good idea, it'll help keep requests as up to date as possible in a way.

**Comment by phillxnet** [comment]
@FroggyFlox I've expanded this issue to cover all our remaining (post chardet's removal) requests secondary dependencies. We don't use any of these directly.

**Comment by phillxnet** [comment]
Closing as:
Fixed by #2589
